scripts/ci-entrypoint.sh

```diff
@@ -91,11 +91,11 @@ create_cluster() {
     # timestamp is in RFC-3339 format to match kubetest
     export TIMESTAMP=$(date -u '+%Y-%m-%dT%H:%M:%SZ')
     ${REPO_ROOT}/hack/create-dev-cluster.sh
+    # export the target cluster KUBECONFIG if not already set
+    export KUBECONFIG="${KUBECONFIG:-${PWD}/kubeconfig}"
```

Comment:
wondering if we should put this outside the create_cluster() func so that your kubeconfig gets auto detected if you're re-running the tests with the cluster from a previous run (ie. SKIP_CREATE is true)

Reply:
here https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/634/files#diff-4bc72f6db52f65045884231f54b3e802L164

Reply:
Yea sounds good.